polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -213,6 +218,23 @@ def get_model_fields(cls) -> list[FieldMeta]:
                         random=cls.__random__,
                     ),
                 )
+        if cls.__set_association_proxy__:
```

Comment:
Nice. Adding as a feature makes sense here. Can appropriate tests and documentation be added for this feature please?

Reply:
Thank! Ok, I'd try to do that.